PowerPointLabs/PowerPointLabs/Models/PowerPointSlide.cs

```diff
@@ -1241,5 +1241,10 @@ public void MakeShapeNamesUnique(Func<Shape, bool> restrictTo = null)
                 currentNames.Add(shape.Name);
             }
         }
+
+        public void DeleteSlideNumberShapes()
+        {
+            DeleteShapesWithPrefix("Slide Number Placeholder");

```

Comment:
This link may help:
https://social.msdn.microsoft.com/Forums/office/en-US/a8a39dd8-be5f-46cf-a86a-a92edc1a064d/powerpoint-identify-shapes-for-footer-information-slide-number-date-footertext?forum=officegeneral